tests/acceptance/blocks/parentruntimeconfig.js

```diff
@@ -0,0 +1,30 @@
+import { ClientFunction } from 'testcafe';
+
+/**
+ * Provides interaction with the runtime config located in the AnswersExperienceFrame
+ */
+class ParentRuntimeConfig {
```

Comment:
This may not be possible, but I'm wondering if we could consolidate down to something like:

``` 
class RuntimeConfigWrapper {
   constructor(runtimeConfig) {
     this.runtimeConfig = runtimeConfig;
   }
   
   async set(key, value) {
      await ClientFunction((key, value) => this.runtimeConfig(key, value));
    }
}
```

Then, instead of having two very similar classes in `ParentRuntimeConfig` and `ChildRuntimeConfig`, we'd have: 
```
const parentRuntimeConfig = new RuntimeConfigWrapper(AnswersExperienceFrame.runtimeConfig);
const childRuntimeConfig = new RuntimeConfigWrapper(AnswersExperience.runtimeConfig);
```

Reply:
Because this is a ClientFunction, I don't think that will work as written because I don't think ClientFunction can be passed a non-serializable parameters such as a function. However, I could pass a boolean flag and toggle the behavior that way to consolidate the RuntimeConfig classes

Reply:
updated to include an `isParent` parameter in the constructor